### PR TITLE
Updated README for Kinothub v1

### PR DESCRIPTION
**Pull request by sumitkolhe**
Updated README to work with Kintohub v1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,34 @@
-# Node sample
+# Node Sample App
 
-A sample node microservice block for [kintohub](http://kintohub.com).
+A sample node microservice block for [Kintohub](http://kintohub.com)
```
Comment (disturbing) [inline README.md:3]: action - Kinto**H**ub

Comment (disturbing) [reply README.md:3]: https://

File: README.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
 
+## :rocket: Deploying on Kintohub v1
 
-# First time setup
+### # Using Dockerfile
```
Comment (disturbing) [inline README.md:7]: action - traditional way should be the only way. remove the dockerfile.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
 
-# Run
+<hr>
 
-* `npm start` run on port 8000
-* `npm run prod` run on port 80
+### # Traditional way
 
-# Support
+- **Type:** Web App
```
Comment (disturbing) [inline README.md:22]: action - Service Type

File: README.md [hunk 4]
```diff
@@ -1,18 +1,34 @@
...
+- **Repo:** https://github.com/kintohub/node-example
+- **Branch:** master
+- **Language:** NodeJS
+- **Language version:** 14
+- **Build Command:** `npm install`
+- **Start Command:** `npm run prod`
+- **Subfolder Path:** `.`
+- **Port:** 80
+
```
Comment (disturbing) [inline README.md:31]: action - add running locally with the commands to start it locally and what you require to install (node, etc)

File: README.md [hunk 5]
```diff
@@ -1,18 +1,34 @@
...
+## :ambulance: Support
 
 http://www.kintohub.com
```
Comment (disturbing) [inline README.md:34]: action - https://www.kintohub.com/contact-us

Add discord link as well: https://discordapp.com/invite/E2CMjKP

**Review by Disturbing — CHANGES_REQUESTED** [review]

**Pull request closed without merging**